rpcs3/Emu/Cell/Modules/cellSaveData.h

```diff
@@ -94,6 +93,13 @@ enum
 	CELL_SAVEDATA_RECREATE_MASK            = 0xffff,
 };
 
+// Dialog Messages
+namespace
```

Comment:
This is horrible. Unnamed namespace in the header. Non-constexpr "constants". And all-caps style.